index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+   
+    <link rel="stylesheet" href="CSS/style.css">
+    <link rel="stylesheet" href="/JavaScript/script.js">
+</head>
+<body>
+
+    <!-- Navigation -->
+
+    <header id="header">
+        <img src="Images/logo.jpg" id="header-img">
+
+        <nav id="nav-bar">
+            <ul>
+                <li><a class="nav-link" href="">Features</a></li>
+                <li><a class="nav-link" href="">About</a></li>
+                <li><a class="nav-link" href="">Contact</a></li>
+            </ul>
+
+
+
+           <!-- <a class="nav-link" href="">Features</a>
+           <a class="nav-link" href="">About</a>
+           <a class="nav-link" href="">Contact</a> -->
```

Comment:
Please remove these empty lines and comments with codes. These are making your code messy.